src/Controller/Component/CrudComponent.php

```diff
@@ -649,7 +650,12 @@ protected function _loadListener($name)
                 throw new ListenerNotConfiguredException(sprintf('Listener "%s" is not configured', $name));
             }
 
+            if (!isset($config['className'])) {
```

Comment:
Seems weird that we'd load the listener if it's not enabled but has configuration. I'm in favor of merging this, as just setting a configuration array doesn't necessarily mean we're loading the thing.

Do we not have some sort of `enabled` key for listeners? I think we do that for actions.